### ISSUE TITLE
Match the accepted "hypervisor" list on DO with what is accepted by BIG-IQ

### ISSUE THREAD

**Issue by Klipope** [F5Networks/f5-declarative-onboarding#216]
### Environment
 * Declarative Onboarding Version: 1.19.0
 * BIG-IP Version: 14.1.4.x

### Summary
DO is restricting the allowed values for the "hypervisor" parameter in BIG-IQ licensing to a specific list. However this list does not match the exact allowed list accepted by BIG-IQ and this causes a DO declaration to not be sent when trying to license a VE in Alibaba. alibaba requires a specific platform-ID to be input in the "hypervisor" parameter field to be interpreted by BIG-IQ.

### Steps To Reproduce
1. Submit a declaration with the hypervisor parameter set to 0x01000028 as defined in the workaround for BIG-IQ bug 949381

2. Observe the following error response:
```json
Thu, 01 Apr 2021 00:55:09 GMT - info: [f5-declarative-onboarding: restWorker.js] Bad declaration: [{"keyword":"additionalProperties","dataPath":".declaration.Common['myLicense']","schemaPath":"#/definitions/regKeyInfo/additionalProperties","params":{"additionalProperty":"bigIqHost"},"message":"should NOT have additional properties"},{"keyword":"enum","dataPath":".declaration.Common['myLicense'].hypervisor","schemaPath":"#/properties/hypervisor/enum","params":{"allowedValues":["aws","azure","gce","vmware","hyperv","kvm","xen"]},"message":"should be equal to one of the allowed values"},{"keyword":"oneOf","dataPath":".declaration.Common['myLicense']","schemaPath":"#/allOf/1/then/oneOf","params":{"passingSchemas":null},"message":"should match exactly one schema in oneOf"}]
```

### Expected Behavior
DO declaration should be able to request a license for a VE in Alibaba Cloud through BIG-IQ

### Actual Behavior
DO declaration is not accepted "Bad declaration" is received when the custom platform-ID is positioned

**Comment by dstokesf5** [comment]
I have added this bug to our internal product backlog as AUTOTOOL-2415. Thank you for reporting it.

**Comment by dstokesf5** [comment]
This issue has been resolved by the 1.22.0 release.
